docs/api-reference/sql-ingestion-api.md

```diff
@@ -299,7 +299,7 @@ The response shows an example report for a query.
         },
         "pendingTasks": 0,
         "runningTasks": 2,
-        "segmentLoadStatus": {
+        "segmentLoadWaiterStatus": {
```

Comment:
This is a correction to the doc right (and not a backward incompatible API change)?

Reply:
Yup, I noticed that the docs were incorrect